src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -23,6 +28,13 @@ def get_attrs(self, item_list, user, **kwargs):
             )
             alert_rule_triggers.append(serialized)
 
+        alert_rule_environments = AlertRuleEnvironment.objects.filter(alert_rule__in=item_list)
```

Comment:
You should use `select_related('environment')` here, otherwise every call to `are.environment` will make a separate query.